pkg/handler/handler.go

```diff
@@ -290,23 +284,24 @@ func (h handler) lastoperation(w http.ResponseWriter, r *http.Request, params ma
 
 	req := broker.LastOperationRequest{}
 
-	queryparams := r.URL.Query()
-
 	// operation is rqeuired
-	if val, ok := queryparams["operation"]; ok {
-		req.Operation = val[0]
+	op := r.FormValue("operation")
+	if op != "" {
+		req.Operation = op
 	} else {
 		h.log.Warning(fmt.Sprintf("operation not supplied, relying solely on the instance_uuid [%s]", instanceUUID))
 	}
 
 	// service_id is optional
-	if val, ok := queryparams["service_id"]; ok {
-		req.ServiceID = val[0]
+	serviceID := r.FormValue("service_id")
+	if serviceID != "" {
+		req.ServiceID = serviceID
 	}
 
 	// plan_id is optional
-	if val, ok := queryparams["plan_id"]; ok {
-		req.PlanID = val[0]
+	planID := r.FormValue("plan_id")
+	if planID != "" {
+		req.PlanID = planID
```

Comment:
Yes, that will work, especially because ```planID``` is not needed, ill update the others as well.